tuneme/templates/new/admin/frontend_users_admin_view.html

```diff
@@ -0,0 +1,89 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n modeladmin_tags %}
+
+{% block titletag %}{{ view.get_meta_title }}{% endblock %}
+
+{% block css %}
+    {{ block.super }}
+    {{ view.media.css }}
+{% endblock %}
+
+{% block extra_js %}
+    {{ view.media.js }}
+{% endblock %}
+
+{% block content %}
+    {% block header %}
+        <header class="nice-padding hasform">
+            <div class="row">
+                <div class="left">
+                    <div class="col">
+                        {% block h1 %}<h1 {% if view.header_icon %}class="icon icon-{{ view.header_icon }}"{% endif %}>{{ view.get_page_title }}<span></span></h1>{% endblock %}
+                    </div>
+                    {% block search %}{% search_form %}{% endblock %}
+                </div>
+                <div class="right">
+                  <form method="post" action="">
+                      {% csrf_token %}
+                        <button type="submit" class="button bicolor icon icon-download">{% trans "Email CSV" %}</button>
```

Comment:
Sorry, I'm a bit confused - should this be wrapped in some logic that checks if a user has those permissions, or will they only ever see this template if they have the prerequisite permissions?